### DISCUSSION TITLE
packages folder not generating

### DISCUSSION THREAD

**Discussion by simplyniceweb** [Laravel-Backpack/community-forum#783] [Q&A (Help)]
Hi,

**Brief info:**
I created a website that runs itself automatically without any interaction from admin/developers.
So i have a shell script that runs some commands including **composer install** and **no errors at all** then that's it site will run and ready to be use.

**Issue:**
My issue about Laravel Backpack is the **public/packages** folder are not being created after running the **composer install** and public/packages folder is added on the .gitignore file.

**Result:**
![image](https://github.com/Laravel-Backpack/community-forum/assets/5621415/7f20728a-7fbd-4428-b052-720f87c4dca1)

**Versions:**
"php": "^8.1",
"laravel/framework": "^10.10",
"backpack/crud": "^5.6",

**Comment by simplyniceweb** [comment]
**Update:**
Removing it from .gitignore file works but i am thinking if there will be a problem in the future if i want to run **composer upgrade**?

**Comment by phpfour** [comment]
Hi @simplyniceweb 

Thanks for submitting your question. 

From Backpack's end, we have to make sure our default setup is compatible with regular use cases, so that majority of the developers would not need to make any change to the way things are set up. However, whenever a project needs something specific the developers should be able to achieve that by themselves. 

To answer your question, I am not sure what you are trying to achieve here so can you please elaborate what exactly are you trying to solve by removing `public/packages` from the `.gitignore` file?

**Comment by jcastroa87** [comment]
Due to not activity i will close this issue, but please feel free to re-open or create a new one if needed.

Cheers.
